scripts/build_framework.sh

```diff
@@ -116,12 +130,23 @@ $LIPO \
   -output "$BOLTS_IOS_BINARY" \
   || die "lipo failed - could not create universal static library"
 
+# Combine AppleTV/Simulator binaries into a single universal binary.
+$LIPO \
```

Comment:
If we are shipping a dynamic framework we probably shouldn't do this, since this is going to end up shipping to everyone. Thoughts?